Appraisals

```diff
@@ -1,43 +1,27 @@
-appraise "rails23" do
-  gem "rails", "~> 2.3.18"
-end
-
-appraise "rails30" do
-  gem "rails", "~> 3.0.20"
-end
-
-appraise "rails31" do
-  gem "i18n", "< 0.7"
-  gem "rails", "~> 3.1.12"
-end
-
-appraise "rails32" do
-  gem "i18n", "< 0.7"
-  gem "rails", "~> 3.2.21"
+appraise "rails50" do
+  gem "rails", "~> 5.0.0"
 end
 
-appraise "rails40" do
-  gem "rails", "~> 4.0.13"
+appraise "rails52" do
+  gem "rails", "~> 5.2.0"
 end
 
-appraise "rails41" do
-  gem "rails", "~> 4.1.16"
+appraise "rails60" do
+  gem "rails", "~> 6.0.0"
 end
```

Comment:
We could consider dropping tests for Rails 5.0 (EOL Apr 2018), 5.2 (EOL Jun 2022) and/or 6.0 (EOL Jun 2023) as well. I kept all versions that worked with JRuby 9.3+.

One reason to be a bit more relaxed with EOLs for JRuby stuff is some of the lag for JRuby itself to catch up with required Ruby versions. Dropping 5.0 is probably pretty reasonable now though.